spec/dummy/config/database.yml

```diff
@@ -2,6 +2,7 @@ default: &default
   adapter: mysql2
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
   timeout: 5000
+  url: <%= ENV["DATABASE_URL"] %>
```

Comment:
Nit: This line can be removed. It should be already handled by rails.

Reply:
Deleted